saleor/graphql/meta/permissions.py

```diff
@@ -118,6 +123,8 @@ def shipping_permissions(_info, _object_pk: Any) -> List[BasePermissionEnum]:
     "App": app_permissions,
     "User": public_user_permissions,
     "Warehouse": product_permissions,
+    "Sale": discount_permissions,
+    "Voucher": discount_permissions,
```

Comment:
Most of these lines (except `App`) are alphabetized, can you enter the word `Sale` in `Voucher` so that they match?